composer.json

```diff
@@ -11,10 +11,11 @@
     }
   , "require": {
         "php": ">=5.4"
-      , "react/socket-client": "^0.5"
-      , "ratchet/rfc6455": "^0.2.1"
+      , "react/socket": "^1.0 || ^0.8 || ^0.7"
+      , "evenement/evenement": "^3.0 || ^2.0"
```

Comment:
Is this new dependency on Evenement related? I don't see this mentioned in the changeset otherwise.

Reply:
This was an existing dependency, when I installed with lowest deps, I found that `WebSocket` was using `EventEmitterTrait`.

Reply:
Indeed! Thanks for spotting and pointing this out, no objections in this case :shipit: :-)